packages/apps/plugins/plugin-table/src/schema/builder.tsx

```diff
@@ -21,49 +21,37 @@ import { getSize } from '@dxos/react-ui-theme';
 
 import { createUniqueProp } from './types';
 
-type TableColumnBuilderOptions = {
+type ColumnCreationOptions = {
   onColumnUpdate?: (id: string, column: ColumnProps) => void;
   onColumnDelete?: (id: string) => void;
   onRowUpdate?: (object: any, key: string, value: any) => void;
   onRowDelete?: (object: any) => void;
 };
 
-/**
- *
- */
-export class TableColumnBuilder {
```

Comment:
These builder classes is a pattern we've used a bunch of other packages and might be good to keep for consistency unless there's a reason it needs to go away

Reply:
In this case the class implementation really obfuscates what is essentially just a call to a pure function.